src/transform/ElementUtilities.js

```diff
@@ -1,4 +1,21 @@
 /**
+ * Polyfill function that tells whether a given element matches a selector.
+ * @param {!Element} el Element
+ * @param {!string} selector Selector to look for
+ * @returns {boolean} Whether the element matches the selector
+ */
+const matchesSelectorCompat = (el, selector) => {
```

Comment:
Sure! I used the `...Compat` nomenclature reflexively because it's very common in the Android ecosystem, but `safeMatches` does sound more friendly.